bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -249,6 +290,18 @@ func (p *packer) encodeUUID(u UUID) {
 	p.putBytes(u[:])
 }
 
+func (p *packer) encodeVersionstamp(v Versionstamp) {
+	p.putByte(versionstampCode)
+
+	if p.versionstampPos != 0 && v.TransactionVersion == incompleteTransactionVersion {
+		panic(fmt.Sprintf("Tuple can only contain one unbound versionstamp"))
```

Comment:
I'm not necessarily opposed to "unbound" as the name for this, but maybe "incomplete" instead so that our usage for what this is called is consistent everywhere?

Additionally, maybe this is more trouble than it's worth, but I suppose we could have the `encode` methods return `error`s, and then we don't need to panic here? It's also possible that only this method and `encodeTuple` need to return `error`s. (We still might decide that we need to panic in `Pack`.)

Reply:
Agree on the naming. I just copy-pasted from another implementation from the bindings from what I recall.